infra_old/s3_dest_bucket.tf

```diff
@@ -0,0 +1,69 @@
+locals {
+  policy_path = "${path.root}/policies"
+  env         = terraform.workspace
+  is_temp     = length(regexall("[a-z]{2,4}-?[0-9]+", local.env)) > 0
+}
+
+resource "aws_s3_bucket" "batch_data_destination_bucket" {
+  bucket        = "immunisation-batch-int-preprod-data-destinations"
```

Comment:
Should this name use `int` or `preprod` rather than both? Same for the data sources bucket